### DISCUSSION TITLE
New version

### DISCUSSION THREAD

**Discussion by luliguachi** [farag2/Sophia-Script-for-Windows#477] [Q&A] [answered]
I have this version of windows, is there any version available?
![Screenshot 2023-04-05 133949](https://user-images.githubusercontent.com/129955720/230069820-264edd38-36a7-4c1b-a07e-bd73359c95ef.png)

**Comment by farag2** [comment]
Just use the latest version. If yiu Windows is out-dated, please run Windows Update and update the system. Very simple.

https://github.com/farag2/Sophia-Script-for-Windows/releases/latest

**Reply by luliguachi** [reply]
I have this problem when using it ![Sophias_script.png](https://user-images.githubusercontent.com/129955720/230102611-d6b0c0d8-e54f-44b2-9334-20e59df794d8.png)

DETAILED: GET https://raw.githubusercontent.com/farag2/Sophia-Script-for-Windows/master/sophia_script_versions.json
with 0-byte payload
DETAILED: received 373-byte response of content type text/plain; charset=utf-8
WARNING: A general error has occurred that is not covered by a more specific error code.
WARNING:
Microsoft Defender corrupted or removed from the operating system

**Comment by farag2** [comment, marked as answer]
Well, you removed Defender by a harmful tweaker (or using a custom broken Window image). That's all. You have to reinstall the OS to be able to use this script. And my duty is to inform you that your system is broken.

**Comment by luliguachi** [comment]
But this is what defender shows me

![imagen_2023-04-05_160009079](https://user-images.githubusercontent.com/129955720/230104095-3cc9ff26-4671-4299-99de-b03b278ac042.png)

**Comment by farag2** [comment]
If script showa that error, it means that something is broken.
Did you apply any tweaks before? And if so, which one?

**Reply by luliguachi** [reply]
I have applied ios bit unninstaler but i have just removed cortana, xbox and that shit, nothing important

**Comment by farag2** [comment]
Do you the original ISO?
When I'll at home I send some commands to test the Defender health.

**Reply by luliguachi** [reply]
Yeah i have the original ISO

**Reply by luliguachi** [reply]
May i restore it from there?

**Comment by farag2** [comment]
Let's wait until I'll be at home in the evening

**Reply by luliguachi** [reply]
Okey at what hour u will he home?

**Comment by farag2** [comment]
In 2 hours. I will ping you. But better PM me in Telegram: @sanctuary_d

**Reply by luliguachi** [reply]
Okey bro
